packages/manager/modules/hub/package.json

```diff
@@ -20,6 +20,7 @@
     "lodash-es": "^4.17.15"
   },
   "peerDependencies": {
+    "@ovh-ux/manager-billing": "^1.0.5",
```

Comment:
👍

Reply:
Thanks to ESLint that has reported me this issue.